fv3config/_datastore.py

```diff
@@ -12,6 +12,8 @@
     USER_CACHE_DIR = os.environ['FV3CONFIG_CACHE_DIR']
 else:
     USER_CACHE_DIR = appdirs.user_cache_dir('fv3gfs', 'vulcan')
+    if not os.path.isdir(USER_CACHE_DIR):
```

Comment:
I would use `os.makedirs(USER_CACHE_DIR, exists_ok=True)` instead of this if statement.